x-pack/plugin/esql/qa/testFixtures/src/main/resources/semantic_text.csv-spec

```diff
@@ -0,0 +1,10 @@
+simple
```

Comment:
Can we add a few more queries with more processing commands here, like eval/where/keep/drop/sort/stats/rename/mv_expand etc? It seems like some of the processing commands might already work out of the box, and having more tests to show the coverage will help to clarify the scope of the PR. #110205 enabled date_nano on a few scalar functions, it seems like we are not there yet, but perhaps it works in a few processing commands already, it will be great if the tests can show more besides `SORT`.

Reply:
added tests for `eval/where/keep/drop/sort/stats/rename/mv_expand`
for functions and operators there is no support - I am doing that as a follow up here https://github.com/ioanatia/elasticsearch/compare/semantic_text_support...ioanatia:elasticsearch:semantic_text_as_string?expand=1
I took the dataset/mapping I had in my follow-up branch and added it here - the dataset has many fields that are not used in the tests yet, but will be used to test things like conversion functions when we add support for those.